src/main/java/vazkii/quark/base/handler/DropoffHandler.java

```diff
@@ -111,11 +112,15 @@ public static boolean isValidChest(EntityPlayer player, IInventory te) {
 			accept = (name.contains("chest") || te instanceof TileEntityChest) && !name.contains("void") && !name.contains("trash");
 		}
 
-		accept = accept && te != null && te.isUsableByPlayer(player);
+		accept = accept && te.isUsableByPlayer(player);
```

Comment:
Was this an intended line change? The TE could still be null here if I've read this correctly.

Reply:
That is incorrect.

"te instanceof IDropoffManager" will only return true if the te is not null, which means it will enter "if (!accept)", where "te.getClass()" is called, which will already crash if the te is null. Thus, by that point in the code, the te has to be non-null or it won't execute.

An explicit null check should probably be added in the beginning if necessary.